tests/test_common.py

```diff
@@ -17,11 +17,14 @@ def test_factorial(self):
         with self.assertRaises(the_exception.InputError):
             common.factorial(56.3)
 
+    # should be removed if our mod function is not in use
+    # some of these 6%3 and 3%6 both fail
```

Comment:
Okay yeah these are the issues that I was talking about. I didn't know how to fix them at the time. I'm okay with removing the tests but we should remove the mod function too then. I added the tests because I didn't want to leave part of the code uncovered and include something broken. If we get rid of the tests without fixing the issue then that defeats the purpose of them.

Reply:
yep, i will remove the function + tests. Just wanted to confirm.

Reply:
Sounds good.